perf/stability/logs-checker/check_k8s_logs.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+
+set -eux
+
+declare -a namespaces=("istio-system" )
+# shellcheck disable=SC2004
+for ((ii=0; ii<15; ii++)) {
+    ns=$(printf 'service-graph%.2d' "${ii}")
+    namespaces+=("${ns}")
+}
+
+function check_events() {
+  printf '\n'
+  if [[ ${#ERRORED[@]} -ne 0 ]]
+  then
+      echo "${#ERRORED[@]} errored pods found."
+      # shellcheck disable=SC2068
+      for CULPRIT in ${ERRORED[@]}
+      do
+        echo "POD: $CULPRIT"
+        echo
+        kubectl get events \
+        --field-selector=involvedObject.name="${CULPRIT}" \
+        -ocustom-columns=LASTSEEN:.lastTimestamp,REASON:.reason,MESSAGE:.message \
+        --all-namespaces \
+        --ignore-not-found=true
+      done
+  else
+      echo "0 pods with errored events found."
+  fi
+}
+
+function check_pod_errors() {
+  # shellcheck disable=SC2068
+  for NAMESPACE in ${namespaces[@]}
+  do
+    echo "Scanning pod logs, Namespace: ${NAMESPACE}"
+    if ! kubectl get ns "${NAMESPACE}";then
+      continue
+    fi
+    while IFS=' ' read -r POD CONTAINERS
+    do
+      for CONTAINER in ${CONTAINERS//,/ }
+      do
+        COUNT=$(kubectl logs --since=24h "${POD}" -c "${CONTAINER}" -n "${NAMESPACE}" | grep -E -c 'error|Error|ERROR|Warn|WARN' || true)
```

Comment:
in fortio client we get errors like this.

```
23:43:38 W http_client.go:697> Parsed non ok code 503 (HTTP/1.1 503)
23:43:39 W http_client.go:697> Parsed non ok code 503 (HTTP/1.1 503)
23:43:41 W http_client.go:697> Parsed non ok code 503 (HTTP/1.1 503)
23:43:41 W http_client.go:697> Parsed non ok code 503 (HTTP/1.1 503)
```
Ok to do it in a follow up PR

Reply:
Additionally this is the error from the cert rotator...
This fails because it cannot find the correct cert.

```
++ openssl s_client -connect 35.239.205.121:443 -servername svc10-0.local -CAfile ca.cert
++ openssl x509 -noout -text
depth=1 O = asm-test-iamwen.svc.id.goog
verify return:1
depth=0 O = Juju org
verify error:num=10:certificate has expired
notAfter=Jul 16 07:08:14 2020 GMT
verify return:1
depth=0 O = Juju org
notAfter=Jul 16 07:08:14 2020 GMT
verify return:1
DONE
++ grep DNS /tmp/aa
                DNS:svc10-0.local, DNS:svc10-0.fake-dns.org, DNS:svc10-0-gen03045.local
+++ grep svc10-0-gen03046.local /tmp/aa
++ [[ -n '' ]]
Did not find svc10-0-gen03046.local
```

Reply:
If fortio gets errors like that, the workload services would also log errors in the mock-service container, right? So I think that's already covered